hasher-matcher-actioner/hmalib/common/configs/tx_collab_config.py

```diff
@@ -94,4 +101,8 @@ def update_collab_config(collab_config: CollaborationConfigBase):
 
     ec.collab_config_class = full_class_name(collab_config.__class__)
     ec.attributes_json_serialized = dataclass_dumps(collab_config)
+
+    if import_as_bank_id:
```

Comment:
nit: slightly confusing, since collab_config is always updated, and import_as_bank_id is only optionally updated. It's documented in the docstring, so ¯\_(ツ)_/¯